apps/src/templates/manageStudents/MoveStudents.jsx

```diff
@@ -107,7 +108,7 @@ class MoveStudents extends Component {
     }),
 
     // redux provided
-    sections: PropTypes.objectOf(
+    sections: PropTypes.arrayOf(
```

Comment:
please add the `hidden` section property here. I think it could be marked required, but ok not to if it breaks things

Reply:
we're filtering out all hidden sections in our redux connection and aren't referencing the `hidden` property in this component, so i don't think this is necessary, but i may be missing something